Configuration/Configurator.h

```diff
@@ -25,7 +25,10 @@
 #define CONFIGURATOR_H_
 
 #include <string>
+#include <set>
+#include <numeric>
 #include <functional>
+#include <boost/format.hpp>
```

Comment:
I'm not in favour of that, especially that there was so much effort to cut down boost to the minimum.
The requested functionality seems entirely possible in standard C++, perhaps a bit less hip but without growing dependencies.

Reply:
OK. It _is_ less hip though.

Reply:
https://github.com/quasar-team/quasar/commit/7b4063ca76b244aa3c8c7906d292619c038108a3